.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
+{"php":"8.1.8","version":"3.4.0:v3.4.0#47177af1cfb9dab5d1cc4daf91b7179c2efe7fad","indent":"\t","lineEnding":"\n","rules":{"encoding":true,"full_opening_tag":true,"blank_line_after_namespace":true,"braces":{"position_after_anonymous_constructs":"same","position_after_control_structures":"same","position_after_functions_and_oop_constructs":"same"},"class_definition":true,"constant_case":true,"elseif":true,"function_declaration":{"closure_function_spacing":"one"},"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ignore"},"no_break_comment":true,"no_closing_tag":true,"no_space_around_double_colon":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":true,"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":{"elements":["property","method","const"]},"align_multiline_comment":true,"array_indentation":true,"binary_operator_spaces":{"default":"single_space"},"blank_line_after_opening_tag":true,"no_unused_imports":true},"hashes":{"tests\/Unit\/Controller\/PageControllerTest.php":2207155758,"tests\/Integration\/AppTest.php":2950225233,"tests\/bootstrap.php":41908297,"appinfo\/routes.php":2458579438,"templates\/adminSettings.php":805129342,"templates\/personalSettings.php":1395204514,"templates\/index.php":2734865279,"node_modules\/flatted\/php\/flatted.php":3670498728,"lib\/Settings\/Admin.php":4177689181,"lib\/Settings\/AdminSection.php":3667266050,"lib\/Settings\/Personal.php":774712035,"lib\/Settings\/PersonalSection.php":237427933,"lib\/Vendor\/Firebase\/JWT\/BeforeValidException.php":3387887423,"lib\/Vendor\/Firebase\/JWT\/Key.php":2086644246,"lib\/Vendor\/Firebase\/JWT\/ExpiredException.php":1091148461,"lib\/Vendor\/Firebase\/JWT\/SignatureInvalidException.php":3046842981,"lib\/Vendor\/Firebase\/JWT\/JWT.php":23558362,"lib\/Vendor\/Firebase\/JWT\/JWK.php":909127881,"lib\/Controller\/LoginController.php":2413542052,"lib\/Controller\/PageController.php":4113731223,"lib\/Controller\/ConfigController.php":2824957717,"lib\/Service\/ProviderService.php":661513495,"lib\/AppInfo\/Application.php":3846310732,"lib\/Backend\/GroupBackend.php":986210557,"lib\/Service\/VirtualOrganisationService.php":3422386287}}
```

Comment:
This file should not be commited to git.
Same thing for the vendor-bin folder.
Maybe add a .gitignore file for this?